apps/yggdrasil/lib/yggdrasil/room.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Yggdrasil.Room do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Yggdrasil.Room
+
+  schema "rooms" do
+    field :title, :string
+    field :description, :string
+    field :is_starting, :boolean, default: false
+    timestamps
+  end
+
+  def create_changeset(params) do
+    %Room{ }
```

Comment:
I should follow the conventions then! I dropped the `model` parameter because it was a create and I was thinking they shouldn't provide one. 

Hey do you remember why we went with a `create_changeset`? Could most things be handled by a regular `changeset` function? I know we discussed this and something came up that led to using multiple changeset functions. I'm thinking for `Room` I might switch this to a single `changeset` since right now there are no differences in context.